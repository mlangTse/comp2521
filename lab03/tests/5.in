16
8
4
2
1
3
6
5
7
12
10
9
11
14
13
15
24
20
18
17
19
22
21
23
28
26
25
27
30
29
31
