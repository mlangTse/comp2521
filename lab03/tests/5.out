Tree:
                              16
                              / \
                             /   \
                            /     \
                           /       \
                          /         \
                         /           \
                        /             \
                       /               \
                      /                 \
                     /                   \
                    /                     \
                   /                       \
                  /                         \
                 /                           \
                /                             \
               /                               \
              8                                24
             / \                               / \
            /   \                             /   \
           /     \                           /     \
          /       \                         /       \
         /         \                       /         \
        /           \                     /           \
       /             \                   /             \
      4              12                 /               \
     / \             / \               /                 \
    /   \           /   \             20                 28
   /     \         /     \           / \                 / \
  2       6       10     14         /   \               /   \
 / \     / \     / \     / \       /     \             /     \
1   3   5   7   9  11   /   \     /       \           /       \
                       13   15   18       22         26       30
                                / \       / \       / \       / \
                               /   \     /   \     /   \     /   \
                              17   19   21   23   25   27   29   31

#nodes = 31,  #leaves = 16
Infix  : 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 
Prefix : 16 8 4 2 1 3 6 5 7 12 10 9 11 14 13 15 24 20 18 17 19 22 21 23 28 26 25 27 30 29 31 
Postfix: 1 3 2 5 7 6 4 9 11 10 13 15 14 12 8 17 19 18 21 23 22 20 25 27 26 29 31 30 28 24 16 
ByLevel: 16 8 24 4 12 20 28 2 6 10 14 18 22 26 30 1 3 5 7 9 11 13 15 17 19 21 23 25 27 29 31 
